content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/clientlibs/site/js/accordion.js

```diff
@@ -0,0 +1,371 @@
+                            navigateAndFocusAccordionItem(index);
+                        });
+                        items[i].addEventListener("keydown", function(event) {
+                            onKeyDown(event);
+                        });
+                    })(i);
+                }
+            }
+        }
+
+        /**
+         * Handles tab keydown events
+         * @private
+         * @param {Object} event The keydown event
+         */
+        function onKeyDown(event) {
+            var index = that._toggle;
+            var lastIndex = that._elementItems.length - 1;
+
+            switch (event.keyCode) {
+                case keyCodes.ARROW_LEFT:
+                case keyCodes.ARROW_UP:
+                    event.preventDefault();
+                    if (index > 0) {
+                        navigateAndFocusAccordionItem(index - 1);
+                    }
+                    break;
+                case keyCodes.ARROW_RIGHT:
+                case keyCodes.ARROW_DOWN:
+                    event.preventDefault();
+                    if (index < lastIndex) {
+                        navigateAndFocusAccordionItem(index + 1);
+                    }
+                    break;
+                case keyCodes.HOME:
+                    event.preventDefault();
+                    navigateAndFocusAccordionItem(0);
+                    break;
+                case keyCodes.END:
+                    event.preventDefault();
+                    navigateAndFocusAccordionItem(lastIndex);
+                    break;
+                default:
+                    return;
+            }
+        }
+
+        /**
+         * Refreshes the accordion item based on the current {@code Accordion#_toggle} index
+         * @private
+         */
+        function refreshAccordion() {
+            if (that._toggle >= 0 && that._elementItempanels && that._elementItems) {
+                if (!that._elementItempanels[that._toggle].classList.contains(selectors.expanded.itempanel)) {
+                    expandAccordionItem(that._elementItempanels[that._toggle], that._elementItems[that._toggle]);
+                } else {
+                    collapseAccordionItem(that._elementItempanels[that._toggle], that._elementItems[that._toggle]);
+                }
+            }
+        }
+
+        /**
+         * Collapses the provided accordion item.
+         * @private
+         * @param {Object} itempanel Provided accordion item's itempanel
+         * @param {Object} item Provided accordion item's item
+         */
+        function collapseAccordionItem(itempanel, item) {
+            if (itempanel && item) {
+                itempanel.classList.remove(selectors.expanded.itempanel);
+                itempanel.setAttribute("aria-hidden", true);
+                item.classList.remove(selectors.expanded.item);
+                item.setAttribute("aria-selected", false);
+                item.setAttribute("tabindex", "-1");
```

Comment:
I'm not sure if the inactive accordion headers should have tabindex set to -1 - they should still be reachable via tab sequence, even if their content is collapsed

Reply:
This should be solved in #601, as we now use `button` controls for which tabindex isn't required.